docs-v2/pages/connected-accounts/oauth-clients.mdx

```diff
@@ -0,0 +1,51 @@
+import Callout from '@/components/Callout'
+import ArcadeEmbed from '@/components/ArcadeEmbed'
+import { Steps } from 'nextra/components'
+
+# OAuth Clients
+By default, most OAuth apps on Pipedream use Pipedream's official OAuth client. That means when you connect an account, you'll see Pipedream's logo and name in the OAuth authorization screen, along with a set of requested permissions (scopes).
+- Pipedream apps solve for a broad range of use cases, which means the scopes requested by Pipedream's official OAuth client may be more than what you need
+- To define the exact scope of access you'd like to grant, you can configure a custom OAuth client for most OAuth apps in Pipedream
```

Comment:
```suggestion
If the scopes are too broad or too narrow for your needs, you can set up your own custom OAuth Client in Pipedream.

When you set up a custom OAuth client for an integrated app in Pipedream, it will replace the Pipedream official OAuth client for that app. This gives you the flexibility to define exactly which scopes should be requested, as well as the opportunity to theme the OAuth client to your brand.
```